src/mavsdk/core/mavlink_message_handler.cpp

```diff
@@ -8,7 +8,16 @@ void MAVLinkMessageHandler::register_one(
 {
     std::lock_guard<std::mutex> lock(_mutex);
 
-    Entry entry = {msg_id, callback, cookie};
+    Entry entry = {msg_id, -1, callback, cookie};
```

Comment:
I think I would prefer to use a `std::optional<uint8_t>` instead of a `int16_t` with `-1` as the magic value.

Reply:
Yes it looks better